src/Tribe/Query.php

```diff
@@ -73,8 +73,18 @@ public static function parse_query( $query ) {
 				$query->set( 'paged', $_REQUEST['tribe_paged'] );
 			}
 
-			// Return early as we don't want to change a post that is not an event.
-			if ( $query->is_main_query() && $query->is_single() && $query->get( 'post_type' ) !== Tribe__Events__Main::POSTTYPE )  {
+			// Return early as we don't want to change a post that is not part of the valid group of event post types.
+			$valid_post_types = [
+				Tribe__Events__Main::POSTTYPE,
+				Tribe__Events__Main::ORGANIZER_POST_TYPE,
```

Comment:
We want to use: `Tribe__Events__Venue::POSTTYPE` and `Tribe__Events__Organizer::POSTTYPE`